force-app/main/default/lwc/marketingCloudCopadoDataTable/marketingCloudCopadoDataTable.js

```diff
@@ -421,10 +423,16 @@ export default class MarketingCloudCopadoDataTable extends LightningElement {
     this.sortData(this.sortedBy, this.sortDirection);
   }
 
-  sortData(fieldname, direction) {
-    
-    let parseData = JSON.parse(JSON.stringify( this.visibleData));
-    // Return the value stored in the field
+  sortData(fieldname, direction) { 
+    let parseData='';
+    // 68 sort the data 
+    if(this.data){
+      parseData = JSON.parse(JSON.stringify(this.data));
+      this.data=null;
+    }else{
```

Comment:
looks to me your eslint/prettier config is not working. can you please run npm install in your project folder?

Reply:
@anil-sfdc ?

Reply:
ok.  using prettier .